easybuild/tools/robot.py

```diff
@@ -359,6 +359,9 @@ def search_easyconfigs(query, short=False, filename_only=False, terse=False):
     search_path = build_option('robot_path')
     if not search_path:
         search_path = [os.getcwd()]
+    extra_search_paths = build_option('search_paths')
```

Comment:
If `--search-paths` is specified, and `search_paths` is registered as a build option, you should be getting the corresponding list of paths here.

Are you saying it's always empty, regardless of the value of `--search-paths`?